lib/ui/discover/discover_child_item.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:farmsmart_flutter/data/model/article_entity.dart';
+import 'package:farmsmart_flutter/ui/myplot/my_plot_item_footer.dart';
+import 'package:farmsmart_flutter/utils/box_shadows.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:farmsmart_flutter/utils/styles.dart';
+import 'package:flutter/material.dart';
+import 'package:farmsmart_flutter/utils/assets.dart';
+
+
+class DiscoveryListItem {
+  Widget buildListItem(ArticleEntity articleData, goToArticleDetail) {
+    return GestureDetector(
+        onTap: () {
+          goToArticleDetail(articleData);
+        },
+        child: Padding(
+            key: ValueKey(articleData.title ?? "No Title"),
+            padding: Margins.boxPadding(),
+            child: Container(
+                height: 90,
+                decoration: BoxDecoration(
+                    color: Colors.white,
+                    borderRadius: BorderRadius.circular(8.0),
+                    boxShadow: BoxShadows.plotListItemShadow()),
+                child: Row(
+                  children: <Widget>[
+                    ClipRRect(
+                      borderRadius: BorderRadius.only(
+                        topLeft: Radius.circular(8.0),
+                        bottomLeft: Radius.circular(8.0),
+                      ),
+//                      child: FadeInImage.assetNetwork(image: cropsData.imagePath, placeholder: 'assets/raw/crop_image_mock.png', fit: BoxFit.cover),
+                      child: FadeInImage.assetNetwork(
+                          image: articleData.imageUrl,
+                          placeholder: Assets.IMAGE_PLACE_HOLDER,
+                          fit: BoxFit.fill),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.only(left: 10),
+                    ),
+                    Expanded(
```

Comment:
I think we should keep, every "child" inside a private method, in the class...
this way you would have like 
Row ( children [
buildTitle,
buildImage,
buildSubtitle
]

all as private members of the class.
I think its a healthy practice, agree?

Reply:
Done !